dotnet/samples/Demos/FunctionInvocationApproval/README.md

```diff
@@ -0,0 +1,44 @@
+# Function Invocation Approval
+
+This console application shows how to use function invocation filter (`IFunctionInvocationFilter`) to invoke Kernel Function only if such operation was approved.
+If function invocation was rejected, the result will contain an information about this, so LLM can react accordingly.
+
+Application uses a sample plugin that allows to build a software by following development stages: collection of requirements, design, implementation, testing and deployment.
```

Comment:
`The application uses a sample plugin which builds software by following these development stages: `